### ISSUE TITLE
Issue: MODULE_NOT_FOUND for an action in Marketplace

### ISSUE THREAD

**Issue by rajyraman** [nektos/act#754]
## System information

    - Operating System: Windows 10
    - Architecture: 64 bit
    - Docker version: 20.10.7, build f0df350
    - act version: 0.2.23

## Expected behaviour

Should run the workflow_dispatch action locally

## Actual behaviour

Getting MODULE_NOT_FOUND error. The action folder with index.js is not in the container.

![image](https://user-images.githubusercontent.com/5035266/126026417-2693c191-648a-43f4-8a21-4cb7af15a737.png)

## Workflow and/or repository

```none
on:
  workflow_dispatch:
    inputs:
      name:
        required: false
        default: World
name: temp
jobs:
  temp:
    runs-on: ubuntu-latest
    env:
      url: https://xxxx.crm6.dynamics.com
      username: actionsuser@xxxx.onmicrosoft.com
      solution: GitHubSolution
    steps:
    - uses: actions/checkout@v2

    - id: who-am-i
      uses: microsoft/powerplatform-actions/who-am-i@main
      with:
        environment-url: ${{ env.url }}
        user-name: ${{ env.username }}
        password-secret: ${{ secrets.password }}
```

</details>

## Steps to reproduce

1. Use workflow with who-am-i action as above
2. Run `act -j temp --use-gitignore false`

## `act` output

<details>
  <summary>Log</summary>

```none

[temp/temp] 🚀  Start image=catthehacker/ubuntu:act-latest
[temp/temp]   🐳  docker pull catthehacker/ubuntu:act-latest
[temp/temp] Removed container: 2175605dc05d8dd783eea77ea0250b966fc720655171e047324f2de6a23d3c2c
[temp/temp]   🐳  docker volume rm act-temp-temp
[temp/temp]   🐳  docker create image=catthehacker/ubuntu:act-latest platform= entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[temp/temp] Created container name=act-temp-temp id=c11c3c07c5aa8efbe2fc0ca87426bb5c4dcca4195e6e7099a3e9ca5f40387fae from image catthehacker/ubuntu:act-latest (platform: )
[temp/temp] ENV ==> [RUNNER_TOOL_CACHE=/opt/hostedtoolcache RUNNER_OS=Linux RUNNER_TEMP=/tmp]
[temp/temp]   🐳  docker run image=catthehacker/ubuntu:act-latest platform= entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[temp/temp] Starting container: c11c3c07c5aa8efbe2fc0ca87426bb5c4dcca4195e6e7099a3e9ca5f40387fae
[temp/temp] Started container: c11c3c07c5aa8efbe2fc0ca87426bb5c4dcca4195e6e7099a3e9ca5f40387fae
[temp/temp]   🐳  docker exec cmd=[mkdir -m 0777 -p /var/run/act] user=root
[temp/temp] Exec command '[mkdir -m 0777 -p /var/run/act]'
[temp/temp]   🐳  docker cp src=/mnt/d/github/powerplatform-actions-private/. dst=/mnt/d/github/powerplatform-actions-private
[temp/temp]   🐳  docker exec cmd=[mkdir -p /mnt/d/github/powerplatform-actions-private] user=
[temp/temp] Exec command '[mkdir -p /mnt/d/github/powerplatform-actions-private]'
[temp/temp] Extracting content from '/tmp/act421746419' to '/mnt/d/github/powerplatform-actions-private'
[temp/temp] Extracting content to '/var/run/act/'
[temp/temp] setupEnv => map[ACT:true AGENT_TOOLSDIRECTORY:/opt/hostedtoolcache CI:true DEPLOYMENT_BASEPATH:/opt/runner GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTION_REF: GITHUB_ACTION_REPOSITORY: GITHUB_ACTOR:nektos/act GITHUB_API_URL:https://api.github.com GITHUB_BASE_REF: GITHUB_ENV:/var/run/act/workflow/envs.txt GITHUB_EVENT_NAME:false GITHUB_EVENT_PATH:/var/run/act/workflow/event.json GITHUB_GRAPHQL_URL:https://api.github.com/graphql GITHUB_HEAD_REF: GITHUB_JOB:temp GITHUB_PATH:/var/run/act/workflow/paths.txt GITHUB_REF:refs/heads/master GITHUB_REPOSITORY:rajyraman/powerplatform-actions-private GITHUB_REPOSITORY_OWNER:rajyraman GITHUB_RETENTION_DAYS:0 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SERVER_URL:https://github.com GITHUB_SHA:0c78094681099dae9ece9cf55cdeef7f1a7f7f47 GITHUB_TOKEN: GITHUB_WORKFLOW:temp GITHUB_WORKSPACE:/mnt/d/github/powerplatform-actions-private IMAGE_OS:ubuntu20 ImageOS:ubuntu20 LSB_RELEASE:20.04 PATH:/opt/hostedtoolcache/node/14.17.1/x64/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin RUNNER_PERFLOG:/dev/null RUNNER_TRACKING_ID: RUNNER_USER:root RUN_TOOL_CACHE:/opt/hostedtoolcache USER:root solution:GitHubSolution url:https://xxxx.crm6.dynamics.com username:actionsuser@xxxx.onmicrosoft.com]
[temp/temp] ⭐  Run actions/checkout@v2
[temp/temp] Skipping local actions/checkout because workdir was already copied
[temp/temp]   ✅  Success - actions/checkout@v2
[temp/temp] setupEnv => map[ACT:true AGENT_TOOLSDIRECTORY:/opt/hostedtoolcache CI:true DEPLOYMENT_BASEPATH:/opt/runner GITHUB_ACTION:who-am-i GITHUB_ACTIONS:true GITHUB_ACTION_REF: GITHUB_ACTION_REPOSITORY: GITHUB_ACTOR:nektos/act GITHUB_API_URL:https://api.github.com GITHUB_BASE_REF: GITHUB_ENV:/var/run/act/workflow/envs.txt GITHUB_EVENT_NAME:false GITHUB_EVENT_PATH:/var/run/act/workflow/event.json GITHUB_GRAPHQL_URL:https://api.github.com/graphql GITHUB_HEAD_REF: GITHUB_JOB:temp GITHUB_PATH:/var/run/act/workflow/paths.txt GITHUB_REF:refs/heads/master GITHUB_REPOSITORY:rajyraman/powerplatform-actions-private GITHUB_REPOSITORY_OWNER:rajyraman GITHUB_RETENTION_DAYS:0 GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SERVER_URL:https://github.com GITHUB_SHA:0c78094681099dae9ece9cf55cdeef7f1a7f7f47 GITHUB_TOKEN: GITHUB_WORKFLOW:temp GITHUB_WORKSPACE:/mnt/d/github/powerplatform-actions-private IMAGE_OS:ubuntu20 INPUT_ENVIRONMENT-URL:https://xxxx.crm6.dynamics.com INPUT_PASSWORD-SECRET: INPUT_USER-NAME:actionsuser@xxxx.onmicrosoft.com ImageOS:ubuntu20 LSB_RELEASE:20.04 PATH:/opt/hostedtoolcache/node/14.17.1/x64/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin RUNNER_PERFLOG:/dev/null RUNNER_TRACKING_ID: RUNNER_USER:root RUN_TOOL_CACHE:/opt/hostedtoolcache USER:root solution:GitHubSolution url:https://xxxx.crm6.dynamics.com username:actionsuser@xxxx.onmicrosoft.com]
[temp/temp] ⭐  Run microsoft/powerplatform-actions/who-am-i@main
[temp/temp]   🐳  docker cp src=/root/.cache/act/microsoft-powerplatform-actions-who-am-i@main/who-am-i/ dst=/var/run/act/actions/microsoft-powerplatform-actions-who-am-i@main/who-am-i/
[temp/temp]   🐳  docker exec cmd=[mkdir -p /var/run/act/actions/microsoft-powerplatform-actions-who-am-i@main/who-am-i/] user=
[temp/temp] Exec command '[mkdir -p /var/run/act/actions/microsoft-powerplatform-actions-who-am-i@main/who-am-i/]'
[temp/temp] Extracting content from '/tmp/act362426294' to '/var/run/act/actions/microsoft-powerplatform-actions-who-am-i@main/who-am-i/'
[temp/temp]   🐳  docker exec cmd=[node /var/run/act/actions/microsoft-powerplatform-actions-who-am-i@main/dist/actions/who-am-i/index.js] user=
[temp/temp] Exec command '[node /var/run/act/actions/microsoft-powerplatform-actions-who-am-i@main/dist/actions/who-am-i/index.js]'
[temp/temp]   | internal/modules/cjs/loader.js:905
[temp/temp]   |   throw err;
[temp/temp]   |   ^
[temp/temp]   | 
[temp/temp]   | Error: Cannot find module '/var/run/act/actions/microsoft-powerplatform-actions-who-am-i@main/dist/actions/who-am-i/index.js'
[temp/temp]   |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
[temp/temp]   |     at Function.Module._load (internal/modules/cjs/loader.js:746:27)
[temp/temp]   |     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
[temp/temp]   |     at internal/main/run_main_module.js:17:47 {
[temp/temp]   |   code: 'MODULE_NOT_FOUND',
[temp/temp]   |   requireStack: []
[temp/temp]   | }
[temp/temp]   ❌  Failure - microsoft/powerplatform-actions/who-am-i@main

```

</details>

**Comment by cmjchrisjones** [comment]
I'm seeing a similar problem using the  https://github.com/actions/setup-dotnet workflow (and at the time of writing GitHub has a major severity issue so I can't do anything so thought I'd turn to try and do something locally)!
 
## System information
  
- Operating System: Windows
- Architecture: x64 (64-bit) 
- Docker version: 20.10.7 
- Docker image used in `act`:
- `act` version: 0.2.6

## Expected behaviour

Successful run



## Actual behaviour

Getting MODULE_NOT_FOUND error. 

```
[.NET/build] Exec command '[node \actions\actions-setup-dotnet@v1\dist\index.js]'
| internal/modules/cjs/loader.js:628
|     throw err;
|     ^
|
| Error: Cannot find module '/github/workspace/\actions\actions-setup-dotnet@v1\dist\index.js'
|     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
|     at Function.Module._load (internal/modules/cjs/loader.js:527:27)
|     at Function.Module.runMain (internal/modules/cjs/loader.js:839:10)
|     at internal/main/run_main_module.js:17:11 {
|   code: 'MODULE_NOT_FOUND',
|   requireStack: []
| }
[.NET/build]   ❌  Failure - Setup .NET
```


## Workflow and/or repository
 

<details>
  <summary>workflow</summary>

```none
name: .NET

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

jobs:
  generateBuildNumber:
    runs-on: ubuntu-latest
    steps:
      - name: Set current date as env variable
        run: echo "NOW::$(date +'%Y-%m-%dT%H:%M:%S')" >> $GITHUB_ENV
      - name: Echo current date
        run: echo $NOW

  build:
    runs-on: ubuntu-latest    
    defaults:
      run:
        working-directory: DemoRazorApp

    steps:
    - uses: actions/checkout@v2
    - name: Setup .NET
      uses: actions/setup-dotnet@v1
      with:
        dotnet-version: 5.0.x
    - name: Restore dependencies
      run: dotnet restore
    - name: Build
      run: dotnet build --no-restore
    - name: Test
      run: dotnet test --no-build --verbosity normal

  tag:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - name: Bump version and push tag
        uses: laputansoft/github-tag-action@v4.6
        with:
          github_token: ${{ env.github-token }}
          tag: ${{ $NOW }}
```

</details>

## Steps to reproduce



## `act` output



<details>
  <summary>Log</summary>

```none
time="2021-08-10T22:45:00+01:00" level=debug msg="Loading workflows from 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.github\\workflows'"
time="2021-08-10T22:45:00+01:00" level=debug msg="Reading workflow 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.github\\workflows\\dotnet.yml'"
time="2021-08-10T22:45:00+01:00" level=debug msg="Reading workflow 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.github\\workflows\\new_release.yml'"
time="2021-08-10T22:45:00+01:00" level=debug msg="Reading workflow 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.github\\workflows\\release.yml'"
time="2021-08-10T22:45:00+01:00" level=debug msg="Using detected workflow event: pull_request"
time="2021-08-10T22:45:00+01:00" level=debug msg="Planning job: build"
time="2021-08-10T22:45:00+01:00" level=debug msg="Loading slug from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:00+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:00+01:00" level=debug msg="Loading revision from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:00+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:00+01:00" level=debug msg="HEAD points to '87ecccea4670378689e43b63483b1889a26a32c3'"
time="2021-08-10T22:45:00+01:00" level=debug msg="HEAD matches \\refs\\heads\\main"
time="2021-08-10T22:45:00+01:00" level=debug msg="using github ref: \\refs\\heads\\main"
[.NET/build] 🚀  Start image=node:12.6-buster-slim
[.NET/build]   🐳  docker pull node:12.6-buster-slim
time="2021-08-10T22:45:00+01:00" level=debug msg="Image exists? true"
[.NET/build]   🐳  docker create image=node:12.6-buster-slim entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[.NET/build] Created container name=act--NET-build id=dec453c011c74b576b514ff950aa04a5b29cab60d99f7d900afecb6759579a0f from image node:12.6-buster-slim
[.NET/build] ENV ==> [RUNNER_TOOL_CACHE=/opt/hostedtoolcache]
[.NET/build]   🐳  docker run image=node:12.6-buster-slim entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[.NET/build] Starting container: dec453c011c74b576b514ff950aa04a5b29cab60d99f7d900afecb6759579a0f
[.NET/build] Started container: dec453c011c74b576b514ff950aa04a5b29cab60d99f7d900afecb6759579a0f
[.NET/build]   🐳  docker cp src=E:\DevWork\BlogPostDemos\GithubActionsDigitalOceanTerraform/. dst=/github/workspace
time="2021-08-10T22:45:02+01:00" level=debug msg="Writing tarball C:\\Users\\Chris\\AppData\\Local\\Temp\\act454906875 from E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform/."
time="2021-08-10T22:45:02+01:00" level=debug msg="Stripping prefix:E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\ src:E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform/."
[.NET/build] Extracting content from 'C:\Users\Chris\AppData\Local\Temp\act454906875' to '/github/workspace'
time="2021-08-10T22:45:06+01:00" level=debug msg="Writing entry to tarball workflow/event.json len:2"
time="2021-08-10T22:45:06+01:00" level=debug msg="Writing entry to tarball home/.act len:0"
[.NET/build] Extracting content to '/github/'
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading slug from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading revision from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD points to '87ecccea4670378689e43b63483b1889a26a32c3'"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD matches \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="using github ref: \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="setupEnv => map[GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:pull_request GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:\\refs\\heads\\main GITHUB_REPOSITORY:cmjchrisjones/blogpostdemo-github-terraform-digitalocean GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:87ecccea4670378689e43b63483b1889a26a32c3 GITHUB_WORKFLOW:.NET GITHUB_WORKSPACE:/github/workspace HOME:/github/home]"
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading slug from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading revision from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD points to '87ecccea4670378689e43b63483b1889a26a32c3'"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD matches \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="using github ref: \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="context env => map[GITHUB_ACTION:0 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:pull_request GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:\\refs\\heads\\main GITHUB_REPOSITORY:cmjchrisjones/blogpostdemo-github-terraform-digitalocean GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:87ecccea4670378689e43b63483b1889a26a32c3 GITHUB_WORKFLOW:.NET GITHUB_WORKSPACE:/github/workspace HOME:/github/home]"
[.NET/build] ⭐  Run actions/checkout@v2
[.NET/build] Skipping actions/checkout
[.NET/build]   ✅  Success - actions/checkout@v2
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading slug from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading revision from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD points to '87ecccea4670378689e43b63483b1889a26a32c3'"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD matches \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="using github ref: \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="setupEnv => map[GITHUB_ACTION:1 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:pull_request GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:\\refs\\heads\\main GITHUB_REPOSITORY:cmjchrisjones/blogpostdemo-github-terraform-digitalocean GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:87ecccea4670378689e43b63483b1889a26a32c3 GITHUB_WORKFLOW:.NET GITHUB_WORKSPACE:/github/workspace HOME:/github/home INPUT_DOTNET-VERSION:5.0.x]"
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading slug from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="Loading revision from git directory 'E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform\\.git'"
time="2021-08-10T22:45:06+01:00" level=debug msg="Found revision: 87ecccea4670378689e43b63483b1889a26a32c3\n"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD points to '87ecccea4670378689e43b63483b1889a26a32c3'"
time="2021-08-10T22:45:06+01:00" level=debug msg="HEAD matches \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="using github ref: \\refs\\heads\\main"
time="2021-08-10T22:45:06+01:00" level=debug msg="context env => map[GITHUB_ACTION:1 GITHUB_ACTIONS:true GITHUB_ACTOR:nektos/act GITHUB_EVENT_NAME:pull_request GITHUB_EVENT_PATH:/github/workflow/event.json GITHUB_REF:\\refs\\heads\\main GITHUB_REPOSITORY:cmjchrisjones/blogpostdemo-github-terraform-digitalocean GITHUB_RUN_ID:1 GITHUB_RUN_NUMBER:1 GITHUB_SHA:87ecccea4670378689e43b63483b1889a26a32c3 GITHUB_WORKFLOW:.NET GITHUB_WORKSPACE:/github/workspace HOME:/github/home INPUT_DOTNET-VERSION:5.0.x]"
[.NET/build] ⭐  Run Setup .NET
[.NET/build]   ☁  git clone 'https://github.com/actions/setup-dotnet' # ref=v1
[.NET/build]   cloning https://github.com/actions/setup-dotnet to C:\Users\Chris\.cache\act/actions-setup-dotnet@v1
[.NET/build] Cloned https://github.com/actions/setup-dotnet to C:\Users\Chris\.cache\act/actions-setup-dotnet@v1
[.NET/build] Checked out v1
time="2021-08-10T22:45:08+01:00" level=debug msg="Read action &{Setup .NET Core SDK GitHub Used to build and publish .NET source. Set up a specific version of the .NET Core CLI in the PATH and set up authentication to a private NuGet repository map[config-file:{Optional NuGet.config location, if your NuGet.config isn't located in the root of the repo. false } dotnet-version:{Optional SDK version to use. If not provided, will install global.json version when available. Examples: 2.2.104, 3.1, 3.1.x false } include-prerelease:{Whether prerelease versions should be matched with non-exact versions (for example 5.0.0-preview.6 being matched by 5, 5.0, 5.x or 5.0.x). Defaults to false if not provided. false } owner:{Optional OWNER for using packages from GitHub Package Registry organizations/users other than the current repository's owner. Only used if a GPR URL is also provided in source-url false } source-url:{Optional package source for which to set up authentication. Will consult any existing NuGet.config in the root of the repo and provide a temporary NuGet.config using the NUGET_AUTH_TOKEN environment variable as a ClearTextPassword false }] map[] {node12 map[] dist/index.js  [] []} {green play}} from 'C:\\Users\\Chris\\.cache\\act\\actions-setup-dotnet@v1\\action.yml'"
time="2021-08-10T22:45:08+01:00" level=debug msg="About to run action &{Setup .NET Core SDK GitHub Used to build and publish .NET source. Set up a specific version of the .NET Core CLI in the PATH and set up authentication to a private NuGet repository map[config-file:{Optional NuGet.config location, if your NuGet.config isn't located in the root of the repo. false } dotnet-version:{Optional SDK version to use. If not provided, will install global.json version when available. Examples: 2.2.104, 3.1, 3.1.x false } include-prerelease:{Whether prerelease versions should be matched with non-exact versions (for example 5.0.0-preview.6 being matched by 5, 5.0, 5.x or 5.0.x). Defaults to false if not provided. false } owner:{Optional OWNER for using packages from GitHub Package Registry organizations/users other than the current repository's owner. Only used if a GPR URL is also provided in source-url false } source-url:{Optional package source for which to set up authentication. Will consult any existing NuGet.config in the root of the repo and provide a temporary NuGet.config using the NUGET_AUTH_TOKEN environment variable as a ClearTextPassword false }] map[] {node12 map[] dist/index.js  [] []} {green play}}"
time="2021-08-10T22:45:08+01:00" level=debug msg="type=3 actionDir=C:\\Users\\Chris\\.cache\\act/actions-setup-dotnet@v1 Workdir=E:\\DevWork\\BlogPostDemos\\GithubActionsDigitalOceanTerraform ActionCacheDir=C:\\Users\\Chris\\.cache\\act actionName=/actions-setup-dotnet@v1 containerActionDir=/actions"
[.NET/build]   🐳  docker cp src=C:\Users\Chris\.cache\act/actions-setup-dotnet@v1 dst=/actions\
time="2021-08-10T22:45:08+01:00" level=debug msg="Writing tarball C:\\Users\\Chris\\AppData\\Local\\Temp\\act354263326 from C:\\Users\\Chris\\.cache\\act/actions-setup-dotnet@v1"
time="2021-08-10T22:45:08+01:00" level=debug msg="Stripping prefix:C:\\Users\\Chris\\.cache\\act\\ src:C:\\Users\\Chris\\.cache\\act/actions-setup-dotnet@v1"
[.NET/build] Extracting content from 'C:\Users\Chris\AppData\Local\Temp\act354263326' to '/actions\'
[.NET/build] Exec command '[node \actions\actions-setup-dotnet@v1\dist\index.js]'
| internal/modules/cjs/loader.js:628
|     throw err;
|     ^
|
| Error: Cannot find module '/github/workspace/\actions\actions-setup-dotnet@v1\dist\index.js'
|     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
|     at Function.Module._load (internal/modules/cjs/loader.js:527:27)
|     at Function.Module.runMain (internal/modules/cjs/loader.js:839:10)
|     at internal/main/run_main_module.js:17:11 {
|   code: 'MODULE_NOT_FOUND',
|   requireStack: []
| }
[.NET/build]   ❌  Failure - Setup .NET
time="2021-08-10T22:45:10+01:00" level=debug msg="exit with `FAILURE`: 1"
time="2021-08-10T22:45:10+01:00" level=debug msg="exit with `FAILURE`: 1"
time="2021-08-10T22:45:10+01:00" level=debug msg="exit with `FAILURE`: 1"
time="2021-08-10T22:45:10+01:00" level=debug msg="exit with `FAILURE`: 1"
Error: exit with `FAILURE`: 1
```

</details>

**Comment by catthehacker** [comment]
I would first recommending upgrading if you are really using `act 0.2.6` since that is super old version and many things has changed since then.

**Comment by catthehacker** [comment]
@rajyraman I found the culprit in the issue you have and already have a fix, just need to figure out if there is any action `microsoft/powerplatform-actions` that we can use as test

**Comment by cmjchrisjones** [comment]
> I would first recommend upgrading if you are really using `act 0.2.6` since that is super old version and many things has changed since then.
> 
> Also I see that you are using `working-directory` option, which was fixed 4 hours ago in master: #772 so maybe check out the artefacts from that pull request and find out if it works for you.

Thanks, I'm not sure why that happened, I only installed it a couple of days ago using chocolatey and that was the version it got for me! I've now ran the chocolatey upgrade and running the latest 0.2.24 release - it appears to be working fine now
